skyreader/plot/plot.py

```diff
@@ -293,60 +381,77 @@ def create_plot(self, result: SkyScanResult, dozoom: bool = False) -> None:
-                vx = np.cos(phi)*np.sin(ra)*np.sin(delta) + np.cos(ra)*(np.cos(delta)*np.sin(dec) + np.cos(dec)*np.sin(delta)*np.sin(phi))
-                vy = np.cos(delta)*np.sin(dec)*np.sin(ra) + np.sin(delta)*(-np.cos(ra)*np.cos(phi) + np.cos(dec)*np.sin(ra)*np.sin(phi))
-                vz = np.cos(dec)*np.cos(delta) - np.sin(dec)*np.sin(delta)*np.sin(phi)
-                idx = healpy.vec2pix(nside, vx, vy, vz)
-                DEC, RA = healpy.pix2ang(nside, idx)
-                Theta[j] = DEC
-                Phi[j] = RA
-            Theta[bins] = Theta[0]
-            Phi[bins] = Phi[0]
-            return Theta, Phi
-
-    def create_plot_zoomed(self,
-                           result: SkyScanResult,
-                           extra_ra=np.nan,
-                           extra_dec=np.nan,
-                           extra_radius=np.nan,
-                           systematics=False,
-                           plot_bounding_box=False,
-                           plot_4fgl=False,
-                           circular=False,
-                           circular_err50=0.2,
-                           circular_err90=0.7):
+        """For plotting circular contours on skymaps ra, dec, sigma all
+        expected in radians."""
+        dec = np.pi/2. - dec
+        sigma = np.rad2deg(sigma)
+        delta, step, bins = 0, 0, 0
+        delta = sigma/180.0*np.pi
+        step = 1./np.sin(delta)/10.
+        bins = int(360./step)
+        Theta = np.zeros(bins+1, dtype=np.double)
+        Phi = np.zeros(bins+1, dtype=np.double)
+        # define the contour
+        for j in range(0, bins):
+            phi = j*step/180.*np.pi
+            comp1 = np.cos(phi)*np.sin(ra)*np.sin(delta)
+            comp2 = np.cos(ra)*np.cos(delta)*np.sin(dec)
+            comp3 = np.cos(dec)*np.sin(delta)*np.sin(phi)
+            comp4 = np.cos(delta)*np.sin(dec)*np.sin(ra)
+            comp5 = np.sin(delta)*np.cos(ra)*np.cos(phi)
+            comp6 = np.sin(delta)*np.cos(dec)*np.sin(ra)*np.sin(phi)
+            comp7 = np.cos(dec)*np.cos(delta)
+            comp8 = np.sin(dec)*np.sin(delta)*np.sin(phi)
+            vx = comp1 + comp2 + comp3
+            vy = comp4 - comp5 + comp6
+            vz = comp7 - comp8
+            idx = healpy.vec2pix(nside, vx, vy, vz)
+            DEC, RA = healpy.pix2ang(nside, idx)
+            Theta[j] = DEC
+            Phi[j] = RA
+        Theta[bins] = Theta[0]
+        Phi[bins] = Phi[0]
+        return Theta, Phi
+
+    def create_plot_zoomed(
+            self,
+            result: SkyScanResult,
+            extra_ra=np.nan,
+            extra_dec=np.nan,
+            extra_radius=np.nan,
+            systematics=False,
+            plot_bounding_box=False,
+            plot_4fgl=False,
+            circular=False,
+            circular_err50=0.2,
+            circular_err90=0.7,
+            angular_error_floor=False,
+            llh_map=True,
+            neutrinofloor_sigma=0.2
```

Comment:
What about merging the `angular_error_floor` with `neutrinofloor_sigma` arguments? You could have it default as `angular_floor_sigma=None`, then passing a value would do apply the smearing later in the function.